AcceptanceTests/DSLExtensions.swift

```diff
@@ -1,9 +1,13 @@
 import Quick
 
 func they(_ description: String, flags: FilterFlags = [:], closure: @escaping () -> Void) {
-    it(description, flags: flags, closure: closure)
+    it("they " + description, flags: flags, closure: closure)
```

Comment:
How did this ever work? And doesn't the output end up as `they it have the description etc etc` if the test fails?

Reply:
also, shouldn't this (and the others in this file) capture file and line metadata?

Reply:
it prints just fine!

![image](https://user-images.githubusercontent.com/7672056/93522607-1ecc9500-f908-11ea-83cb-f97fab134d72.png)

as for the file + line, I think quick handles it internally somehow...even the original impl doesnt have it, different from custom XCTAssert family which requires it to function properly.

Reply:
weeeird 😆 and those red lines show up in the right place in the editor?

Reply:
it does! maybe because quick generates the test cases dynamically? (I think it does at least)